inc/drivers/cutebot/CutebotMotor.h

```diff
@@ -0,0 +1,29 @@
+#ifndef CUTEBOT_MOTORS_H
+#define CUTEBOT_MOTORS_H
+
+#include "MicroBitComponent.h"
+#include "MicroBitI2C.h"
+#include "inc/drivers/Motor.h"
+
+enum CutebotMotorId {
+  CUTEBOT_MOTOR_ID_RIGHT = 1,
+  CUTEBOT_MOTOR_ID_LEFT,
+};
+
+class CutebotMotor : public Motor
+{
+    MicroBitI2C i2c;
```

Comment:
Should this be `&i2c`?

Reply:
I think you're right. I'm not sure how it compiled like that. (And worked for the GigglebotMotor?)